examples/classification/main.py

```diff
@@ -453,7 +453,7 @@ def train_epoch(train_loader, model, criterion, criterion_fn, optimizer, compres
             config.tb.add_scalar("train/top1", top1.avg, i + global_step)
             config.tb.add_scalar("train/top5", top5.avg, i + global_step)
 
-            for stat_name, stat_value in compression_ctrl.statistics(quickly_collected_only=True).items():
+            for stat_name, stat_value in compression_ctrl.statistics(quickly_collected_only=True).as_dict().items():
```

Comment:
Can the `tb.add_scalar` handle nested dicts? Because your `as_dict` structure is, in general, nested.

Reply:
The `tf.add_scalar` is not support nested dicts.